baselines/baseline_template/.gitignore

```diff
@@ -0,0 +1,2 @@
+*__pycache__/
+*.DS_Store
```

Comment:
Should this be a full Python `.gitignore`? Or can we remove it because it's redundant with the root `.gitignore`?